services/reference/linea/json-rpc-methods/eth_call.mdx

```diff
@@ -1,40 +1,14 @@
 ---
 title: "eth_call"
+hide_title: true
+hide_table_of_contents: true
```

Comment:
The first description for entire blockId is lost, works in RPC playground: 
<img width="857" alt="image" src="https://github.com/user-attachments/assets/96a13561-d80d-4752-8168-8f685ce300bb">

Reply:
Pulled, did install, built about 10 mins before this comment